packages/daemon/bootstrap.js

```diff
@@ -81,17 +91,21 @@ function createPidfileSync(pathname) {
     });
     // emit `server` event in app
     app.emit('server', server);
-
+  
     // server listen
     await new Promise(resolve => {
-      server.listen(PORT, resolve);
+      if (HOST) {
```

Comment:
```suggestion
      if (typeof HOST === 'string') {
```

Reply:
Actually if the host is number 127.1, it works well on macOS and linux :)
If it must be a string, I think it's better to check when we read it from config file.